string name
bool value
int32 priority
float32 deadline
